src/Symfony/Component/Serializer/Normalizer/PropertyNormalizer.php

```diff
@@ -150,4 +152,27 @@ protected function setAttributeValue($object, $attribute, $value, $format = null
 
         $reflectionProperty->setValue($object, $value);
     }
+
+    /**
+     * @param string|object $classOrObject
+     * @param string        $attribute
+     *
+     * @return \ReflectionProperty
+     *
+     * @throws \ReflectionException
+     */
+    private function getReflectionProperty($classOrObject, $attribute)
+    {
+        $class = is_string($classOrObject) ? $classOrObject : get_class($classOrObject);
```

Comment:
This is not needed as the `ReflectionClass` constructor takes either a class or an object as an argument.